code/modules/antagonists/cult/runes.dm

```diff
@@ -289,6 +289,11 @@ structure_check() searches for nearby cultist structures required for the invoca
 		convertee.say("You son of a bitch! I'm in.", forced = "That son of a bitch! They're in. (April Fools)")
 
 	else
+		// Remove other stuns, includes cult hand's secondary effects
+		convertee.AdjustAllImmobility(0 SECONDS)
```

Comment:
`AdjustAllImmobility(0)` = "Reduce all stuns by 0 seconds". You want `SetAllImmobility`